### PR TITLE
chore(Storybook): New UI for Storybook

### PR DESCRIPTION
**Pull request by yceballost**

### CODE REVIEW

File: .storybook/manager.js [hunk 1]
```diff
@@ -6,4 +6,7 @@ import {createStorybookTheme} from './storybook-manager-theme';
 addons.setConfig({
     panelPosition: 'right',
     theme: createStorybookTheme('Movistar'),
+    toolbar: {
+        title: {hidden: true},
```
Comment (yceballost) [inline .storybook/manager.js:10]: This hides this element
Screenshot 2023-10-26 at 18 09 52

File: .storybook/manager-head.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<style type="text/css">
+    /* styles for the storybook sidebar */
+    .search-result-item {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.1) !important;
```
Comment (atabel) [inline .storybook/manager-head.html:4]: are all those `!important` needed? There isn't any official way of customizing these storybook styles?

I'm afraid a future update of storybook could break this

File: .storybook/manager-head.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    /* icon size */
+    .sidebar-svg-icon,
+    .sidebar-item svg,
+    .search-result-item svg {
+        width: 4px !important;
+        height: 4px !important;
+        color: inherit !important;
+        opacity: 0 !important; /* hide icon to keep aligment with the rest of list elements */
```
Comment (atabel) [inline .storybook/manager-head.html:36]: `visibility: hidden`?

File: .storybook/manager-head.html (hunk 2 above)
Comment (yceballost) [reply .storybook/manager-head.html:4]: I followed the example from NASA storybook
https://github.com/nasa-jpl/explorer-1/blob/main/storybook/manager-head.html

https://storybook.js.org/docs/react/configure/theming#css-escape-hatches

File: .storybook/manager-head.html (hunk 3 above)
Comment (yceballost) [reply .storybook/manager-head.html]: I solved the aligment by other way. Hiding icons with display none and adding some margins-right

File: .storybook/manager-head.html (hunk 2 above)
Comment (yceballost) [reply .storybook/manager-head.html]: I removed `!important` tag from the most of cases. The cases with `!important` needs it to force override the styles :(

**Review by atabel — APPROVED** [review]

**Pull request merged**